changelogs/unreleased/8193-add-new-database-connection-pool-setting.yml

```
@@ -3,18 +3,15 @@ issue-nr: 8193
 change-type: major
 destination-branches: [master]
 sections:
-  upgrade-note: >
-    Previously, a single database connection pool was shared among the server and the agent, configured
-    via the `database.connection_pool_max_size` and `database.connection_pool_min_size` options.
-    For each environment, the new resource scheduler uses a database connection pool of its own.
+  upgrade-note: |
+    Previously, a single database connection pool configured
+    via the {inmanta.config:option}`database.connection_pool_max_size` and {inmanta.config:option}`database.connection_pool_min_size` options.
+    Now, the new resource scheduler uses an additional database connection pool per environment.
     Please review the following connection pool options and make sure they fit your
     needs. Do keep in mind that these options will apply to all active environments:
-      - For the server: [:inmanta.config:option:`server.db_connection_pool_max_size`, :inmanta.config:option:`server.db_connection_pool_min_size`, :inmanta.config:option:`server.db_connection_timeout]
-        By default, the new server pool will be 50% the size of the previous global pool.
-      - For the scheduler: ["scheduler.db_connection_pool_max_size", "scheduler.db_connection_pool_min_size", "scheduler.db_connection_timeout"]
+      - For the server: {inmanta.config:option}`server.db_connection_pool_max_size`, {inmanta.config:option}`server.db_connection_pool_min_size`, {inmanta.config:option}`server.db_connection_timeout
+        By default, the server pool will be 50% the size of the previous global pool.
+      - For the scheduler: {inmanta.config:option}`scheduler.db_connection_pool_max_size`, {inmanta.config:option}`scheduler.db_connection_pool_min_size`, {inmanta.config:option}`scheduler.db_connection_timeout`
         By default, each scheduler pool will be 10% the size of the previous global pool.
+    These defaults are tuned to 5 environments per server, if you have more or less, please adjust accordingly. 
```

```suggestion
    These defaults are tuned to 5 environments per server, if you have more or fewer, please adjust accordingly. 
```
I think, could be wrong.